packages/protocol/contracts/common/FeeCurrencyWhitelist.sol

```diff
@@ -46,7 +53,51 @@ contract FeeCurrencyWhitelist is
    * @return Patch version of the contract.
    */
   function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
-    return (1, 1, 1, 0);
+    return (1, 1, 2, 0);
+  }
+
+  // removed with address zero
+  function setAdaptor(address token, address adaptor) external onlyOwner {
```

Comment:
We should also be able to remove Adapter

Reply:
you can set it to zero

Reply:
it is not the same though ... it will not remove it from the storage